x-pack/plugin/mapper-constant-keyword/src/internalClusterTest/java/org/elasticsearch/xpack/constantkeyword/mapper/ConstantKeywordFieldMapperTests.java

```diff
@@ -203,4 +206,24 @@ protected void randomFetchTestFieldConfig(XContentBuilder b) throws IOException
     protected boolean allowsNullValues() {
         return false;   // null is an error for constant keyword
     }
+
```

Comment:
We have enough test cases that have to implement these four identical 'empty' methods, that maybe it's worth consolidating them into a `NoSyntheticSourceTest` interface with default methods and the test cases can just implement them?